### ISSUE TITLE
output figures display at terminal

### ISSUE THREAD

**Issue by crazyhottommy** [shenwei356/csvtk#6]
Hi shenwei,

Thanks for this nice tool. I really like the plot functions.

It will be more useful if it can be piped to `display` from  imagemagic to display on terminal rather saving the figures. see some examples http://crazyhottommy.blogspot.com/2014/11/csvkit-to-manipulate-csv-at-command.html.  Most of the time, I just want to glance the distribution of the data. If I want serious figures, I will import to R and plot with ggplot2.

Best,
Tommy

**Comment by shenwei356** [comment]
Hi Tommy,

Output image file could be set by global flag `-o/--out-file` of `csvtk` .

Help message of `csvtk plot`:

```shell
plot common figures

Notes:

  1. File format is determined by the out file (-o/--out-file) suffix.
     Supported formats: eps, jpg|jpeg, pdf, png, svg, and tif|tiff

Usage:
  csvtk plot [command]

Available Commands:
  box         plot boxplot
  hist        plot histogram
  line        line plot and scatter plot

Flags:
      --axis-width float     axis width (default 1.5)
  -f, --data-field string    column index or column name of data (default "1")
  -g, --group-field string   column index or column name of group
      --height float         Figure height (default 4.5)
      --label-size int       label font size (default 14)
      --tick-width float     axis tick width (default 1.5)
      --title string         Figure title
      --title-size int       title font size (default 16)
      --width float          Figure width (default 6)
      --x-max string         maximum value of X axis
      --x-min string         minimum value of X axis
      --xlab string          x label text
      --y-max string         maximum value of Y axis
      --y-min string         minimum value of Y axis
      --ylab string          y label text
```

**Comment by crazyhottommy** [comment]
I want to do something like:

```bash
csvtk -t plot hist grouped_data.tsv.gz -f2 -o - | display
display: no decode delegate for this image format `' @ error/constitute.c/ReadImage/501.

```

**Comment by shenwei356** [comment]
I've finished it,  and I'll release a new version when I fix a bug of other subcommand.

**Comment by crazyhottommy** [comment]
great and thanks!

**Comment by shenwei356** [comment]
Implemented in new version [v0.4.2](https://github.com/shenwei356/csvtk/releases/tag/v0.4.2)

**Comment by crazyhottommy** [comment]
nicely worked for me! great.
